applications/crossbar/doc/voicemail.md

```diff
@@ -48,7 +48,7 @@ Key | Description | Type | Default | Required | Support Level
 `skip_greeting` | Determines if the greeting should be skipped | `boolean()` | `false` | `false` | `supported`
 `skip_instructions` | Determines if the instructions after the greeting and prior to composing a message should be played | `boolean()` | `false` | `false` | `supported`
 `timezone` | The default timezone | `string(5..32)` |   | `false` | `supported`
-`transcribe` | Transcribe voicemail using ASR engine | `boolean()` | `false` | `false` | `alpha`
+`transcribe` | Transcribe voicemail using ASR engine | `boolean()` | `false` | `false` | `supported`
```

Comment:
This should be alpha, and is already fixed in the master, remove latest commit and rebase to master